readme.txt

```diff
@@ -0,0 +1,71 @@
+=== wp-cldr ===
```

Comment:
AFAIK yes it has to be a separate file and follow a fairly strict format so that the plugin repo can pull all the data in for display.